src/lib/tunes/tune_definition.desc

```diff
@@ -0,0 +1,108 @@
+/****************************************************************************
```

Comment:
@dagar Do you know if this will show up in https://px4.github.io/Firmware-Doxygen ?
If not, is there anything we can do to make that happen?